plugins/chat/test/javascripts/acceptance/hashtag-css-generator-test.js

```diff
@@ -3,9 +3,14 @@ import { visit } from "@ember/test-helpers";
 import { test } from "qunit";
 
 acceptance("Chat | Hashtag CSS Generator", function (needs) {
-  const category1 = { id: 1, color: "ff0000" };
-  const category2 = { id: 2, color: "333" };
-  const category3 = { id: 4, color: "2B81AF", parentCategory: { id: 1 } };
+  const category1 = { id: 1, color: "ff0000", name: "category1" };
+  const category2 = { id: 2, color: "333", name: "category2" };
+  const category3 = {
+    id: 4,
+    color: "2B81AF",
+    parentCategory: { id: 1 },
+    name: "category3",
+  };
```

Comment:
@martin-brennan Faking category objects is making the test case fragile. I tried relying on client side fixtures but that is hard as well. Perhaps we want to reconsider writing this as a system test instead?